Code/Editor/Lib/Tests/test_ModularViewportCameraController.cpp

```diff
@@ -427,4 +446,58 @@ namespace UnitTest
         // Clean-up
         HaltCollaborators();
     }
+
+    TEST_F(ModularViewportCameraControllerFixture, CameraModifersAreDetectedWhenViewportIsNotInFocus)
+    {
+        SandboxEditor::SetCameraCaptureCursorForLook(false);
+
+        const float deltaTime = 1.0f / 60.0f;
+
+        // Given
+        PrepareCollaborators();
```

Comment:
Given what?

Reply:
This is following the ['Given/When/Then'](https://martinfowler.com/bliki/GivenWhenThen.html) test structure and this is just to signify the different parts of the test (similar to 'Arrange/Act/Assert'). This is a convention we use a lot throughout quite a few viewport tests so I'd like to leave this in